gematria/granite/graph_builder_model_inference.cc

```diff
@@ -373,6 +407,12 @@ GraphBuilderModelInference::FromTfLiteModel(
         llvm::errc::invalid_argument,
         "The special token index tensor could not be read");
   }
+
+  // Get the set of annotation names used by the model.
+  llvm::Expected<std::set<std::string>> annotation_names =
+      GetAnnotationNames(**interpreter);
```

Comment:
I'm a bit worried here - requiring that the list of annotations is always present will invalidate all the existing tflite files (not that there are too many). Maybe we should have a flag allowing the load the older models that do not have annotations.